app/src/main/java/com/gh4a/utils/HttpImageGetter.java

```diff
@@ -538,32 +535,25 @@ private Bitmap renderSvgToBitmap(Resources res, InputStream is) {
                 int docWidth = (int) (svg.getDocumentWidth() * density);
                 int docHeight = (int) (svg.getDocumentHeight() * density);
                 if (docWidth < 0 || docHeight < 0) {
+                    // The image doesn't have a specified width or height, so we try to determine
+                    // its aspect ratio to render it with correct proportions
                     float aspectRatio = svg.getDocumentAspectRatio();
-                    if (aspectRatio > 0) {
-                        float heightForAspect = (float) mMaxWidth / aspectRatio;
-                        float widthForAspect = (float) mMaxHeight * aspectRatio;
-                        if (widthForAspect < heightForAspect) {
-                            docWidth = Math.round(widthForAspect);
-                            docHeight = mMaxHeight;
-                        } else {
-                            docWidth = mMaxWidth;
-                            docHeight = Math.round(heightForAspect);
-                        }
+                    if (aspectRatio != -1) {
+                        // we make it smaller than maxWidth so that it doesn't take too much screen space
+                        docWidth = Math.round(mMaxWidth * 0.7f);
```

Comment:
Right, I should have explained it in the PR description 😅
The previous logic ended up setting `docWidth` to `mMaxWidth / 2` in these cases, because `docWidth` was initially set to `mMaxWidth` and then divided by 2 in the while loop below since `docWidth` wasn't `< mMaxWidth`.
The `mMaxWidth * 0.7` is an arbitrary choice I've made to leverage a bit more screen space than `mMaxWidth / 2`, and at the same time to avoid using an excessive amount of space (which would happen if we used `mMaxWidth` directly).

Also, if we revert the [above change](https://github.com/slapperwan/gh4a/pull/1230#discussion_r961362156), not using `mMaxWidth` as the `docWidth` becomes more important because it would prevent a bit of image border from being cut off.

Reply:
Reverted